presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchErrorCode.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.elasticsearch;
+
+import com.facebook.presto.spi.ErrorCode;
+import com.facebook.presto.spi.ErrorCodeSupplier;
+import com.facebook.presto.spi.ErrorType;
+
+import static com.facebook.presto.spi.ErrorType.EXTERNAL;
+
+public enum ElasticsearchErrorCode
+        implements ErrorCodeSupplier
+{
+    ELASTIC_SEARCH_CLIENT_ERROR(0, EXTERNAL),
+    ELASTIC_SEARCH_MAPPING_REQUEST_ERROR(1, EXTERNAL),
+    ELASTIC_SEARCH_EXCEEDS_MAX_HIT_ERROR(2, EXTERNAL);
+
+    private final ErrorCode errorCode;
+
+    ElasticsearchErrorCode(int code, ErrorType type)
+    {
+        errorCode = new ErrorCode(code + 0x0503_0000, name(), type);
```

Comment:
why `503`? Is it unique?

Reply:
yep, following other connector's error code, need a unique number for Elasticsearch, now 503 is unique